src/main/java/seedu/address/storage/JsonSerializableTruthTable.java

```diff
@@ -9,73 +9,73 @@
 import com.fasterxml.jackson.annotation.JsonRootName;
 
 import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.AddressBook;
-import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.ReadOnlyTruthTable;
+import seedu.address.model.TruthTable;
 import seedu.address.model.person.Person;
 import seedu.address.model.team.Team;
 
 /**
- * An Immutable AddressBook that is serializable to JSON format.
+ * An Immutable TruthTable that is serializable to JSON format.
  */
 @JsonRootName(value = "addressbook")
```

Comment:
Should this be addressbook? Can check compatability first.
```suggestion
@JsonRootName(value = "truthtable")
```